packages/utilities/src/dateTimeFormat/absolute-test.js

```diff
@@ -54,6 +89,26 @@ describe(locale, () => {
         expect(actualOutput).toBe(expectedOutput);
       });
 
+      describe('with timeZone', () => {
+        timeZones.forEach((timeZone) => {
+          const dtf = new Intl.DateTimeFormat(locale, {
+            timeStyle,
+            dateStyle: timeStyle,
+            timeZone,
+          });
+          const expectedOutput = dtf.format(date);
+
+          test(`${timeStyle} in ${timeZone} → ${expectedOutput}`, () => {
+            const actualOutput = absolute.format(date, {
+              locale,
+              style: timeStyle,
+              timeZone,
+            });
+            expect(actualOutput).toBe(expectedOutput);
+          });
+        });
+      });
+
```

Comment:
A couple of consistency nitpicks:

1. In `formatTime()` and `formatDate()`, the time zone `test()`s are top-level.
2. In `format()` and `formatRange()`, the time zone `test()`s are wrapped in `describe()` blocks.